src/main/resources/fileTemplates/internal/d_language_enum.d.ft

```diff
@@ -0,0 +1,6 @@
+module ${DLANGUAGE_MODULE_NAME};
+
+#include("module ${DLANGUAGE_MODULE_NAME};.ft")
```

Comment:
this doesn't look right. perhaps a find/replace mistake

Reply:
Yes, it was an attempt to mudodularize existing templates, unfortunately couldn't find how to allow includes in velocity templates in Idea. Would be glad if someone knows how to enable them

Reply:
I think it's been attempted in the past. So is this pull request still a work in progress? `"module ${DLANGUAGE_MODULE_NAME};.ft"` will not find a valid template

Reply:
Not a work in progress. Just missed it while reverting back to non-modularized version.